barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/ultra_circuit_builder.hpp

```diff
@@ -482,7 +482,10 @@ class UltraCircuitBuilder_ : public CircuitBuilderBase<typename Arithmetization_
     uint32_t put_constant_variable(const FF& variable);
 
   public:
-    size_t get_num_constant_gates() const override { return 0; }
+    size_t get_num_constant_gates() const override
+    {
+        return 0;
+    }
```

Comment:
```suggestion
    size_t get_num_constant_gates() const override { return 0; }
```

reverting unwanted formatting change